doc/content/devices/abp-vs-otaa/_index.md

```diff
@@ -0,0 +1,53 @@
+---
+title: "ABP vs OTAA"
+description: ""
+---
+
+This section can help you understand the differences between ABP and OTAA activation modes for LORaWAN end devices, and comprehend why using OTAA is recommended. 
+
+<!--more-->
+
+A `DevEUI` is a 64-bit unique ID assigned to an end device by the manufacturer. This value is linked to the hardware and it cannot be altered.
+
+Unlike `DevEUI`, which identifies an end device globally, a 32-bit `DevAddr` identifies the end device within the current network and all communication after joining the network is done with it. A `DevAddr` value consists of `NwkID` and `NwkAddr`. `NwkID` is a 7-bit network identifier, used to distinguish (overlapping) LoRaWAN networks, while `NwkAddr` is a 25-bit network address of the end device.
+
+{{< note >}} `DevAddr` are not unique identifiers. If there are more than one device with identical `DevAddr`, {{% tts %}} Network Server has to perform **matching**. See more in the [Network Server reference]({{< ref "/reference/components/network-server#uplink-handling" >}}) section. {{</ note >}}
+
+A `DevAddr` and security keys are assigned to an end device during a procedure called **activation**. LoRaWAN supports two modes of activating an end device:
+
+- ABP (Activation By Personalization)
+- OTAA (Over-The-Air Activation)
```

Comment:
I don't think we like such bullet lists, maybe just pull them into the paragraph above.